packages/api-utils/lib/getSlug.js

```diff
@@ -4,14 +4,17 @@ const require = createRequire(import.meta.url);
 
 const { slugify } = require("transliteration");
 
+const standardSlug = "a-zA-Z0-9-";
+
 /**
  * @name getSlug
  * @summary Return a slugified string using "slugify" from transliteration
  * @see https://www.npmjs.com/package/transliteration
  * @memberof Utils
  * @param {String} slugString - string to slugify
+ * @param {String|Boolean} allowedChars - specify extra characters that are not removed by slugify
```

Comment:
The official JSDocs syntax for optional parameter with default value:

```suggestion
 * @param {String} [allowedChars=a-zA-Z0-9-_.~'] - specify extra characters that are not removed by slugify
```